### ISSUE TITLE
Investigate using plausible for analytics

### ISSUE THREAD

**Issue by plaindocs** [writethedocs/www#1871]
instead of GA

CC @ericholscher @mxsasha

**Comment by ericholscher** [comment]
I'm :+1: on this, or we could just remove analytics? I think the only thing we really use it for is the annual stats heh.

**Comment by mxsasha** [comment]
Haven't tried it but looks good, I've also seen matamo.org as a GA alternative. Seems both are $19/mo at our size, may be not worth it for just the stats post. Don't we also use the numbers for ad sales though?

**Comment by plaindocs** [comment]
I feel that some kind of analytics would is good for things like job-board? 

I'm -1 on Matomo after using it at $oldjob

**Comment by ericholscher** [comment]
Yea, plausible is pretty nice. It's basic, but does the job and easy to use. We're using it on RTD and I like it. 

In terms of the website sponsorship, we do actually get view stats from the ads, which is another way we could get some of this data, but that doesn't include conf sites, etc.

I'm not sure if the job board would support third-party JS integration. I do know it has native GA support, because everything does 🙃

**Comment by plaindocs** [comment]
Yeah, I think it would be interesting to track some really basic page views, and see how we go. 
One of us could look at basic reports, or we could ask @goatlady to cook some up sporadically. 

Could try it for six months and turn it off if it's not useful?
